.ko.yaml

```diff
@@ -0,0 +1 @@
+defaultBaseImage: registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
should we use a different image here 🙏 ?

Reply:
done :+1: